test/banana.test.js

```diff
@@ -4,6 +4,9 @@ import Banana from '../src'
 import assert from 'assert'
 import fs from 'fs'
 import path from 'path'
+import semver from 'semver'
+
+const isNodeVersionAbove12 = () => semver.major(process.version) > 12
```

Comment:
Should be declared as dev-dependency in package.json. Or you could rather just use `parseInt(process.versions.node) > 12`

Reply:
ok, done